lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2112,6 +2117,10 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
     return mapValue(BV, Call);
   }
 
+  case OpAsmCallINTEL:
```

Comment:
I don't see handling of `OpAsmTargetINTEL` in the SPIR-V reader, is this intentional?

Reply:
Yes. SPIRVAsmTargetINTEL is a SPIRVEntry, not a SPIRVValue, so it cannot be handled here. Instead it is managed by SPIRVAsmINTEL